### DISCUSSION TITLE
fragment identifier is not defined error in ePub Checker

### DISCUSSION THREAD

**Discussion by eBookAuthor** [apprenticeharper/DeDRM_tools#2100] [Q&A]
Hi, 

I just revised an ePub file and ran it through pagina ePub Checker which alerted me to the error: 

fragment identifier is not defined

I then opened the ePub file in SIGIL and went to check the line where ePub checker found the error, where I found this code:

<p class="ind" id="chap5_ft2"><a id="chap5_ftg"></a>Although he was then still largely unknown...

Can anyone reading that suggest a way I might remove the "fragment identifier is not defined" error from that code?  

Thanks!

**Comment by ElleKayEm** [comment]
I suggest asking the folks at MobileRead.

**Comment by eBookAuthor** [comment]
Thanks! I have posted there and already got a handful of helpful replies!
Great suggestion...
Best...

    On Friday, July 22, 2022 at 02:01:29 AM PDT, ElleKayEm ***@***.***> wrote:  
 
 


I suggest asking the folks at MobileRead.
